python/plugins/processing/tests/testdata/qgis_algorithm_tests.yaml

```diff
@@ -122,3 +122,57 @@ tests:
         compare:
           geometry:
             precision: 7
+
+  # Eliminate sliver polygons
+  # case 1: merge with largest area
+  - algorithm: qgis:eliminatesliverpolygons
+    name: Eliminate sliver polygons largest area
+    params:
+      ATTRIBUTE: '"fid"'
+      COMPARISON: '0'
+      COMPARISONVALUE: '"polys.5"'
+      INPUT:
+        name: polys.shp
+        type: vector
+      KEEPSELECTION: 'False'
+      MODE: '0'
+    results:
+      OUTPUT:
+        name: expected/eliminate_largest_area.gml
+        type: vector
+
+  # case 2: merge with smallest area
+  - algorithm: qgis:eliminatesliverpolygons
+    name: Eliminate sliver polygons smallest area
+    params:
+      ATTRIBUTE: '"fid"'
```

Comment:
Can you replace this with

    ATTRIBUTE: 'fid'

and

      COMPARISONVALUE: 'polys.5'

I'll fix the test creator script accordingly